ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -193,4 +286,52 @@
         }
     });
 
+    /**
+     * whenever RTE value of enumName is changed, we save it's value corresponding to text value field.
+     */
+    channel.on("change", "div[data-wrapperclass='cmp-adaptiveform-base__richTextEnumNames']", function (e) {
+        Coral.commons.ready(e.target, function(component) {
+            var i = $(component.parentNode.parentNode.parentNode.parentNode.parentNode).index(),
+                richTextValue = channel.find(BASE_WRAPPER_RICH_TEXT_ENUMNAMES),
+                textValue = channel.find(BASE_ENUMNAMES_VISIBLE);
+            if(textValue[i] && richTextValue[i]){
+                changeTextValue(textValue[i], richTextValue[i]);
+            }
+        });
+    });
+
+    /**
+     * whenever RTE value of richTextTitle is changed, we save it's value to corresponding text value field.
+     */
+    channel.on("change", "div[data-wrapperclass='cmp-adaptiveform-base__richtexttitle']", function (e) {
```

Comment:
Why do you need div here ? Generally data attributes is for JS mapping, please try to use other CSS selector here

Reply:
same as above